x-pack/plugins/apm/public/components/app/service_overview/service_overview_instances_table/index.tsx

```diff
@@ -0,0 +1,275 @@
+import { px, unit } from '../../../../style/variables';
+import { ServiceOverviewTableContainer } from '../service_overview_table_container';
+import { ServiceNodeMetricOverviewLink } from '../../../shared/Links/apm/ServiceNodeMetricOverviewLink';
+import { MetricOverviewLink } from '../../../shared/Links/apm/MetricOverviewLink';
+
+type ServiceInstanceItem = ValuesType<
+  APIReturnType<'GET /api/apm/services/{serviceName}/service_overview_instances'>
+>;
+
+interface Props {
+  serviceName: string;
+}
+
+export function ServiceOverviewInstancesTable({ serviceName }: Props) {
+  const { agentName, transactionType } = useApmServiceContext();
+
+  const {
+    urlParams: { start, end },
+    uiFilters,
+  } = useUrlParams();
+
+  const columns: Array<EuiBasicTableColumn<ServiceInstanceItem>> = [
+    {
+      field: 'name',
+      name: i18n.translate(
+        'xpack.apm.serviceOverview.instancesTableColumnNodeName',
+        {
+          defaultMessage: 'Node name',
+        }
+      ),
+      render: (_, item) => {
+        const { serviceNodeName } = item;
+        const isMissing = serviceNodeName === SERVICE_NODE_NAME_MISSING;
+        const text = isMissing
+          ? UNIDENTIFIED_SERVICE_NODES_LABEL
+          : serviceNodeName;
+
+        const link =
+          agentName === 'java' ? (
+            <ServiceNodeMetricOverviewLink
+              serviceName={serviceName}
+              serviceNodeName={item.serviceNodeName}
+            >
+              {text}
+            </ServiceNodeMetricOverviewLink>
+          ) : (
+            <MetricOverviewLink
+              serviceName={serviceName}
+              mergeQuery={(query) => ({
+                ...query,
+                kuery: isMissing
+                  ? `NOT (service.node.name:*)`
+                  : `service.node.name:"${item.serviceNodeName}"`,
+              })}
+            >
+              {text}
+            </MetricOverviewLink>
+          );
+
+        return <TruncateWithTooltip text={text} content={link} />;
+      },
+      sortable: true,
+    },
+    {
+      field: 'latencyValue',
+      name: i18n.translate(
+        'xpack.apm.serviceOverview.instancesTableColumnLatency',
+        {
+          defaultMessage: 'Latency',
+        }
+      ),
+      width: px(unit * 10),
+      render: (_, { latency }) => {
+        return (
+          <SparkPlotWithValueLabel
+            color="euiColorVis1"
+            series={latency?.timeseries}
+            valueLabel={asDuration(latency?.value)}
+          />
+        );
+      },
+      sortable: true,
+    },
+    {
+      field: 'throughputValue',
+      name: i18n.translate(
+        'xpack.apm.serviceOverview.instancesTableColumnThroughput',
+        {
+          defaultMessage: 'Traffic',
+        }
+      ),
+      width: px(unit * 10),
+      render: (_, { throughput }) => {
+        return (
+          <SparkPlotWithValueLabel
```

Comment:
tangential: `SparkPlotWithValueLabel` seems like it should simply be:
```ts
<SparkPlot />
```

Then, if we want to hide the label value:
```ts
<SparkPlot showValueLabel={false} />
```